### PR TITLE
Update continuous-profiling GA for TiDB 6.0.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilyjazz** [comment]
@breeswish @crazycs520 PTAL

**Review by breeswish — APPROVED** [review]

File: dashboard/continuous-profiling.md [hunk 1]
```diff
@@ -17,16 +13,16 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 使用持续性能分析时，应留意如下使用限制：
 
-- 该功能可在 x86 架构下支持 TiDB、PD、TiKV、TiFlash；而在 ARM 框架下还未完全兼容，不可开启。
+- 该功能可在 x86 架构下支持 TiDB、PD、TiKV；而在 ARM 框架下还未完全兼容，不可开启。
```
Comment (shichun-0415) [inline dashboard/continuous-profiling.md:16]:
```suggestion
- 该功能可在 x86 架构下支持 TiDB、TiKV 和 PD；而在 ARM 框架下，该功能还未完全兼容，不可开启。
```

File: dashboard/continuous-profiling.md [hunk 2]
```diff
@@ -17,16 +13,16 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
...
 
 - 该功能适用于使用 v1.9.0 及以上版本 TiUP 或 v1.3.0 及以上版本 TiDB Operator 部署和升级的集群，不支持二进制包部署和升级的集群。
 
 ## 分析内容
 
-持续性能分析允许用户在不重启的情况下持续收集 TiDB、TiKV、PD、TiFlash 各个实例的性能数据，并且持久监控节点。收集到的性能数据可显示为火焰图、有向无环图等，直观展现实例在性能收集的时间段内执行的各种内部操作及其比例，方便用户快速了解该实例 CPU 资源消耗细节。目前支持的性能信息：
+持续性能分析允许用户在不重启的情况下持续收集 TiDB、TiKV、PD 各个实例的性能数据，并且持久监控节点。收集到的性能数据可显示为火焰图、有向无环图等，直观展现实例在性能收集的时间段内执行的各种内部操作及其比例，方便用户快速了解该实例 CPU 资源消耗细节。目前支持的性能信息：
```
Comment (shichun-0415) [inline dashboard/continuous-profiling.md:22]:
```suggestion
持续性能分析允许用户在不重启的情况下持续收集 TiDB、TiKV 和 PD 各个实例的性能数据，并且持久监控节点。收集到的性能数据可显示为火焰图、有向无环图等，直观展现实例在性能收集的时间段内执行的各种内部操作及其比例，方便用户快速了解该实例 CPU 资源消耗细节。目前支持的性能信息：
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**